libs/layoutlib/build.gradle

```diff
@@ -1,61 +0,0 @@
-import org.ajoberstar.grgit.Grgit
```

Comment:
Only used here, there are traces of it left in toml and build-logic.

Also "closes #884"

Reply:
Removed the leftovers 👍 [e62b010](https://github.com/cashapp/paparazzi/pull/1228/commits/e62b010f7a167a8d923f259a3b89fbcccfac35e6)

Reply:
Nice, also just realised, transitively "fixes #717"